subworkflows/local/build_references/main.nf

```diff
@@ -64,14 +64,14 @@ workflow BUILD_REFERENCES {
 
     def ch_hgnc_date = Channel.empty()
     def ch_hgnc_ref  = Channel.empty()
-//TODO: unify as if(tools.contains("fusioninspector")) once nextflow bug fixed
+    //TODO: unify as if(tools.contains("fusioninspector")) once nextflow bug fixed
     def run_fusioninspector = tools.contains("fusioninspector")
-    if(run_fusioninspector) {
+    if(run_fusioninspector && !params.skip_vcf) {
```

Comment:
Would it make sense to have a dependency matrix in one place instead of spreading out the logic in multiple if statements that are harder to maintain? If you agree, we can open an issue (not for this PR)

Reply:
Yes I would love that! But that seems really complicated with this pipeline, so might be better for a next release?